host/xtest/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ set (SRC
 	xtest_test.c
 	xtest_uuid_helpers.c
 )
+if(CFG_SECURE_PARTITION AND CFG_SPMC_TESTS)
+	list (APPEND SRC regression_9000.c)
```

Comment:
I think it would make sense to have this in a separate test suite, compare with the PKCS#11 suite below. It could still be run by default if enabled as we do for the GP tests.

Reply:
Ack